insalan/pizza/admin.py

```diff
@@ -0,0 +1,28 @@
+from django.contrib import admin
+
+from .models import Pizza, TimeSlot, Order, PizzaOrder
+
+class PizzaAdmin(admin.ModelAdmin):
+    list_display = ("id", "name", "ingredients")
+    search_fields = ["name", "ingredients"]
```

Comment:
Maybe being able to search by allergens would be useful too

Reply:
Good idea